k8s-fluentbit/main.tf

```diff
@@ -0,0 +1,31 @@
+locals {
+  # this should be the default calculated name anyway, but we want to enforce it just to be sure
+  service_account_name = trimprefix(var.release == trimsuffix(var.release, "-fluentbit") ? "${var.release}-fluentbit": var.release, "-")
```

Comment:
This logic is much more complex than the other modules:
https://github.com/massdriver-cloud/terraform-modules/blob/main/k8s-ingress-nginx/main.tf#L3

We should simplify it and be consistent with how the other modules do it.  The point here is so EKS/GKE/AKS can run it and just call it `fluentbit` like a native installation, but if/when we bundle-ize this, we can use `md_metadata.name_prefix` as the release and it will work as a custom installation.